infra/modules/azure_monorepo_single_env_starter_pack/rbac_ad.tf

```diff
@@ -0,0 +1,65 @@
+# subscription roles defined in `eng-azure-authorization` repo
+
+resource "azurerm_role_assignment" "admins_group_rg" {
+  scope                = azurerm_resource_group.main.id
+  role_definition_name = "Owner"
+  principal_id         = var.entraid_groups.admins_object_id
```

Comment:
maybe we can have descriptions as well?

Reply:
Makes sense to me, both for TF description as well as improving clarity when browsing permission on Azure Portal